nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/resources/docs/org.apache.nifi.snmp.processors.GetSNMP/additionalDetails.html

```diff
@@ -54,16 +54,18 @@ <h2>Configuration Details</h2>
     <li><b>Host</b> - [REQUIRED] the name of the host where the SNMP agent is running. (default is localhost)</li>
     <li><b>Port</b> - [REQUIRED] the port number on which the SNMP agent is listening. (default is 161)</li>
     <li><b>OID</b> - [REQUIRED] the root OID to request.</li>
+    <li><b>SNMP Strategy</b> - [REQUIRED] SNMP Get or SNMP Walk request strategy.</li>
     <li><b>Version</b> - [REQUIRED] the SNMP version to use (SNMPv1 [default], or SNMPv2c, or SNMPv3)</li>
     <li><b>Community</b> - [REQUIRED if SNMPv1 or SNMPv2c] the SNMP community to use. (default is public)</li>
     <li><b>Security Level</b> - [REQUIRED if SNMPv3] the security level to use. (default is authPriv)</li>
     <li><b>Security Name</b> - [REQUIRED if SNMPv3] the security name (user name) to use.</li>
     <li><b>Authentication Protocol</b> - [REQUIRED if SNMPv3 and authNoPriv or authPriv] the authentication protocol to use.</li>
     <li><b>Authentication Password</b> - [REQUIRED if SNMPv3 and authNoPriv or authPriv] the authentication password to use.</li>
-    <li><b>Private Protocol</b> - [REQUIRED if SNMPv3 and authPriv] the private protocol to use.</li>
-    <li><b>Private Password</b> - [REQUIRED if SNMPv3 and authPriv] the private password to use.</li>
-    <li><b>Retries</b> - [REQUIRED] Sets the number of retries to be performed before a request is timed out. (default is 0)</li>
-    <li><b>Timeout</b> - [REQUIRED] Sets  timeout in milliseconds before a confirmed request is resent or timed out. (default is 5000)</li>
+    <li><b>Privacy Protocol</b> - [REQUIRED if SNMPv3 and authPriv] the privacy protocol to use.</li>
+    <li><b>Privacy Password</b> - [REQUIRED if SNMPv3 and authPriv] the privacy password to use.</li>
+    <li><b>Retries</b> - [NOT REQUIRED] Sets the number of retries to be performed before a request is timed out. (default is 0)</li>
+    <li><b>Timeout</b> - [NOT REQUIRED] Sets  timeout in milliseconds before a confirmed request is resent or timed out. (default is 1000)</li>
+    <li><b>Textual OID</b> - [NOT REQUIRED] Optionally added to the flow file</li>
```

Comment:
Are these the same properties as the PropertyDescriptors in the processor?
If yes, the same names should be used as on the UI (eg. Password vs Passphrase).
I feel a bit overkill to maintain this list (unless it contains additional info, but I don't really see it).

Reply:
I feel likewise, I'll remove the list of properties.